core.spec.md

```diff
@@ -320,7 +364,7 @@ The prefix MUST NOT be elided within documentation; definitions of schema elemen
 Feature specs MUST prefix the following schema elements:
   - the names of any object types, interfaces, unions, enums, or input object types defined by the feature
   - the names of any fields the spec introduces on *foreign types* defined in a different feature
```

Comment:
Not sure we should keep this bullet and the next one (anything that mentions a "foreign" spec basically), because 1) I don't think the spec is currently clear on how such a thing should work/be valid and 2) I'm not sure how to even make that work with the whole "you can just copy-paste" definitions.

Tbc, I'm sure we'll have needs for combining specs somewhat at some point, but it'll be easier to think that through when we have concrete use cases, and it's easier to add new capability later than to remove them.

If we do remove this, we should also update the example of that section.

Reply:
Removed these lines.